bootstrap/service.go

```diff
@@ -42,6 +42,7 @@ var (
 	errCheckChannels      = errors.New("failed to check if channels exists")
 	errConnectionChannels = errors.New("failed to check channels connections")
 	errUpdateCert         = errors.New("failed to update cert")
+	errDecodeExtKey       = errors.New("failed to decode external key")
```

Comment:
I don't see this error is added to the HTTP API error handler. You can either wrap it with `ErrMalformed` or export it and handle it in the HTTP error handler.